hack/templates/values/values.template

```diff
@@ -441,6 +441,20 @@ nicFeatureDiscovery:
   #       cpu: "300m"
   #       memory: "150Mi"
 
+docaTelemetryService:
+  deploy: false
+  image: {{ .DOCATelemetryService.Image }}
+  repository: {{ .DOCATelemetryService.Repository }}
+  version: {{ .DOCATelemetryService.Version }}
+  # containerResources:
```

Comment:
nit: add image pull secrets comment like above

Reply:
Can do - but the helm part of this isn't working right now. I need to remove helm changes in this PR and add it as a follow-up.

Reply:
It should be done now - this version of the PR has helm support.